src/components/autocomplete/js/listItemDirective.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  'use strict';
+  angular
+      .module('material.components.autocomplete')
+      .directive('mdListItem', MdListItem);
+
+  function MdListItem ($compile, $mdUtil) {
+    return {
+      require: '^mdAutocomplete',
```

Comment:
Does this share any functionality with the `mdItem` which already exists as a child of `mdList`? We talked about renaming `mdItem` to `mdListItem` for consistency. Would that cause a conflict with this directive?